app/views/lightweight_activities/new.html.haml

```diff
@@ -13,43 +13,60 @@
       %li / New Activity
 
 
-%h1 New activity
-= form_for @activity, :url => '/activities' do |f|
-  = f.error_messages
-  = edit_menu_for(@activity, f)
-  =# f.hidden_field 'investigation_id', :value =>@activity.investigation.id # LightweightActivities currently don't belong_to Investigations
-  - if defined?(session[:user_id])
-    = f.hidden_field 'user_id', :value => session[:user_id]
-  = field_set_tag 'Activity Name' do
-    = f.text_field :name
-  = field_set_tag 'Theme' do
-    = f.select :theme_id, options_from_collection_for_select(Theme.all, 'id', 'name'), { :include_blank => "None (inherit from sequence, project, or site default)" }
-  - if current_user.is_admin
-    = field_set_tag 'Project' do
-      = f.select :project_id, options_from_collection_for_select(Project.all, 'id', 'title', @activity.project_id), { :include_blank => true }
-    = field_set_tag "Official CC Activity" do
-      = f.check_box :is_official
-    = field_set_tag "Is this activity locked?" do
-      = f.check_box :is_locked
-      ("Locked" activities may only be copied or edited by admin users, not regular authors.)
-  = field_set_tag 'Publication status' do
-    = f.select :publication_status, options_for_select(LightweightActivity::PUB_STATUSES_OPTIONS, @activity.publication_status)
-  = field_set_tag 'Runtime environment' do
-    = f.select :runtime, options_for_select(LightweightActivity::RUNTIME_OPTIONS, @activity.runtime)
-    .hint (Once the activity is published, you can't change this setting. You'll have to make a copy and change it for the copy instead.)
-  = field_set_tag 'Activity Description' do
-    = f.text_area :description, :cols => 80, :rows => 5, :class => 'wysiwyg'
-  = field_set_tag 'Related Content' do
-    = f.text_area :related, :cols => 80, :rows => 5, :class => 'wysiwyg'
-  = field_set_tag 'Estimated time to complete, in minutes' do
-    = f.text_field :time_to_complete
-  = field_set_tag 'Thumbnail URL' do
-    = f.text_field :thumbnail_url
-  = field_set_tag 'Activity layout' do
-    = f.select :layout, options_for_select(LightweightActivity::LAYOUT_OPTIONS, @activity.layout)
-  - if current_user.is_admin
-    = field_set_tag "Authoring Mode" do
-      = f.select :editor_mode, options_for_select(LightweightActivity::EDITOR_MODE_OPTIONS, @activity.editor_mode)
-  = field_set_tag 'Notes' do
-    The notes area is for storing information for other authors, e.g. source material or usage restrictions, and is not intended to be displayed in the runtime.
-    = f.text_area :notes
+%h1 New Activity
+.sequence_form
+  = form_for @activity, :url => '/activities' do |f|
+    = f.error_messages
+    =# f.hidden_field 'investigation_id', :value =>@activity.investigation.id # LightweightActivities currently don't belong_to Investigations
```

Comment:
We could remove the above line that is commented out, obviously this isn't your edit, just legacy stuff in here.